discord/bot.go

```diff
@@ -45,6 +45,9 @@ var LobbyUpdateChannels = make(map[string]*chan LobbyStatus)
 
 var ChannelsMapLock = sync.RWMutex{}
 
+var privateChannelIDTemp = "";
```

Comment:
Having this in the global space represents a race condition. If the bot is added to 2 guilds before one calls `.au new` (which is guaranteed to happen on startup), then one will override the other's tempchannelID. Not sure why this needs to be in the global spaces regardless? Do you need a temp channel ID for any particular reason?

Reply:
The reason I put a temporary variable here is that I am unfamiliar with how I can access guildStateData from this line of code:
https://github.com/CloakingPotion/amongusdiscord-1/blob/private-prompt/discord/bot.go#L505

In this method: 'newGuild' the .json configuration file is loaded and it's only here where I have access to it. But I'm not sure how to access the guildState data, so I set up a temporary variable that I can update in that method and use when .au new is called.

And to be honest, I didn't even think of the possibility that it can be connected to two guilds at the same time.

How would you suggest I access the guildState information from the newGuild method? Thanks for the help so far!

Reply:
Oh so PersistentGuildData is a field/variable in the GuildState struct. So anytime we have a GuildState instance `guild`, we can call `guild.PersistentGuildData` to access it like any other variable in a struct. If that makes sense.

Reply:
I see. I was able to get the guildState from bot.AllGuilds[guildId]. Thank you! :D